elasticsearch5/docker/my_init.d/99-generate-ssl

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
Could we use a different init number here so we don't overwrite the base image behaviour?
They could survive in parallel quite happily.